gnovm/pkg/gnolang/op_assign.go

```diff
@@ -1,7 +1,40 @@
 package gnolang
 
-func (m *Machine) doOpDefine() {
-	s := m.PopStmt().(*AssignStmt)
+import (
+	"fmt"
+
+	"github.com/gnolang/gno/tm2/pkg/errors"
+)
+
+var ErrOpNotSupported = errors.New("operation not supported")
+
+var assignOps map[Op]func(s *AssignStmt, m *Machine) (cpuCycles int64) = map[Op]func(s *AssignStmt, m *Machine) int64{
```

Comment:
maps are expensive and `Op` is a `uint8`, so change to map to `[256]func(...) ...`?